features/perpetual/contract-state/GeneralInfo.tsx

```diff
@@ -0,0 +1,191 @@
+import styled from "styled-components";
```

Comment:
this file is directly copied from Emp contract-state/GeneralInfo. no review required really